model/task/task.go

```diff
@@ -1001,6 +1010,24 @@ func (t *Task) markAsHostDispatchedWithFunc(doUpdate func(update bson.M) error,
 	return nil
 }
 
+// newTaskOutput returns a new task output with the most up-to-date
+// configuration for the task run. This function should only be used to set the
+// task output field upon task dispatch.
+func (t *Task) newTaskOutput() *taskoutput.TaskOutput {
```

Comment:
So I originally had it that, but thought it was misleading because it is not exactly just "getting" the task output, rather it is "initializing" it or "setting" it. Maybe `initializeTaskOutput` or `populateTaskOutput` would be more appropriate?